Directory.Build.props

```
@@ -1,31 +1,34 @@
 <Project>
-  <PropertyGroup>
-    <_ParentDirectoryBuildPropsPath Condition="'$(_DirectoryBuildPropsFile)' != ''">$([System.IO.Path]::Combine('..', '$(_DirectoryBuildPropsFile)'))</_ParentDirectoryBuildPropsPath>
-  </PropertyGroup>
+    <PropertyGroup>
+        <UseArtifactsOutput>true</UseArtifactsOutput>
+        <ArtifactsPath>$(MSBuildThisFileDirectory)artifacts</ArtifactsPath>
 
-  <Import Project="$(_ParentDirectoryBuildPropsPath)" Condition="Exists('$(_ParentDirectoryBuildPropsPath)')"/>
+        <LangVersion>latest</LangVersion>
+        <Features>strict</Features>
+        <DebugType>embedded</DebugType>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
 
-  <PropertyGroup>
-    <WarningLevel>3</WarningLevel>
-    <AnalysisLevel>preview</AnalysisLevel>
-    <Features>strict</Features>
-  </PropertyGroup>
+        <EnableNETAnalyzers>true</EnableNETAnalyzers>
+        <AnalysisLevel>latest</AnalysisLevel>
+        <AnalysisMode>recommended</AnalysisMode>
+        <IsAotCompatible>true</IsAotCompatible>
+        <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+        <ReportAnalyzer>true</ReportAnalyzer>
 
-  <PropertyGroup>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-  </PropertyGroup>
+        <ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">true</ContinuousIntegrationBuild>
+        <!-- TODO re-enable -->
+        <!-- <TreatWarningsAsErrors Condition="'$(ContinuousIntegrationBuild)' == 'true' or '$(Configuration)' == 'Release'">true</TreatWarningsAsErrors> -->
 
-  <PropertyGroup>
-    <IsPackable>true</IsPackable>
-  </PropertyGroup>
+        <Authors>David Pine</Authors>
+        <Copyright>2022 David Pine</Copyright>
```

This doesn't match the originals:
```suggestion
        <Copyright>© 2022-20224 David Pine</Copyright>
```

---

I've changed this on purpose to conform with the [REUSE/SPDX convention](https://reuse.software/faq/#copyright-symbol).
Feel free to revert.